pkg/resourceprovider/resourceprovider.go

```diff
@@ -116,6 +116,9 @@ func (resourceProvider *ResourceProvider) StartMineLoop(ctx context.Context) cha
 
 	taskCh := make(chan Task)
 	resourceProvider.controller.web3Events.Pow.SubscribenewPowRound(func(newPowRound pow.PowNewPowRound) {
+
+		PostCard()
```

Comment:
check the return error.  a better way use fmt.Errorf in PostCard function.  and check the return error and log it.

Reply:
Added error checking and logging.